about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formula 1 Get Together Conference</title>
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <header class="hamburger-logo">
+
+        <div class="header-container">
+            <div><a href="#"><img src="images/mobile_menu1.svg" class="ham1"></a></div>
+            <nav>
+                <img src="images/F1_logo.png" class="logo">
+            </nav>
+        </div>
+    </header>
+
+    <nav class="menu" id="menu-pop">
+        <img src="images/Icon-close.svg" alt="" class="close-button">
+        <ul class="nav-content">
+            <li><a href="about.html">About</a></li>
```

Comment:
[REQUIRED] When we navigate to the About page we don't have a link to go back to the homepage. Kindly replace the About link with a Home link

Reply:
Resolved the issue